gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -7145,6 +7176,20 @@ properties:
         $ref:
           - "_terms_enum.yaml#/type_2/common"
 
+  pediatric_kidney_staging:
+    $ref:
+      - "_terms.yaml#/pediatric_kidney_staging/common"
```

Comment:
I don't think there is a corresponding entry in `_terms.yaml` for `pediatric_kidney_staging` which may be the reason the build is failing

Reply:
@wwysoc2 and @mpsolano I think that's my bad - I can update that... was supposed to do that on Wednesday.